src/config/config.ts

```diff
@@ -1,9 +1,12 @@
 import { ENV } from '@/lib/constants'
 import { Chain, defineChain } from 'viem'
-import { rootstock, rootstockTestnet } from 'viem/chains'
-import { createConfig, http } from 'wagmi'
+import { WagmiAdapter } from '@reown/appkit-adapter-wagmi'
+import { rootstock, rootstockTestnet } from '@reown/appkit/networks'
+import { createConfig, http, cookieStorage, createStorage } from 'wagmi'
 import { injected } from 'wagmi/connectors'
 
+export const REOWN_PROJECT_ID = process.env.NEXT_PUBLIC_REOWN_PROJECT_ID as string
```

Comment:
I'd suggest to move this to constants file as we're centralizing all env values ​​then export from there.